### DISCUSSION TITLE
Openlogin Adapter Whitelabeling contrast problem

### DISCUSSION THREAD

**Discussion by kubrysh** [Web3Auth/Web3Auth#522] [Feature Requests]
In case when whitelabeling OpenLogin Adapter with options: `dark: true`, and setting the primary color to a light one (e.g. `theme: { primary: '#FFFF00' }`) there are some contrast problems (see attached screenshots).

My suggestion is to implement switching to dark text / icon background when light primary colors are used for whitelabeling OpenLogin Adapter.

<details>
  <summary>Screenshots</summary>

  ![screenshot-1](https://user-images.githubusercontent.com/55651856/187889076-c7e0cfef-3bf9-4a85-8dae-18a0cf9e5e59.jpg)
  ![screenshot-2](https://user-images.githubusercontent.com/55651856/187889107-f232b9ab-76f6-4b7b-b0e9-fc6cc2944f88.jpg)
  ![screenshot-3](https://user-images.githubusercontent.com/55651856/187889581-96b0d62f-109b-4fc8-ba40-d41bed565ca2.jpg)
</details>

**Comment by YZhenY** [comment]
@itsinz @lionellbriones

**Comment by itsinz** [comment]
Hey @kubrysh, apologies for the delayed response. Thank you for flagging this out. 
We will be working on allowing more customization on the text and icon background to cater to contrast cases like this. 
Let us update you when this is up!

**Reply by kubrysh** [reply]
Thanks, [itsinz](https://github.com/itsinz)! Looking forward for any updates on addressing this issue.
